www/app/page5.controller.js

```diff
@@ -12,18 +12,18 @@ module.exports = function ($scope) {
     return hash;
   }
 
+  $scope.currentroom = document.domain;
   $scope.nick = "Anon";
-  //$scope.currentroom = document.domain;
   $scope.rooms = [];
   $scope.messages = [];
   $scope.newMessage = {};
 
   //For localhost testing
   if (document.domain == 'localhost') {

```

Comment:
Why is this needed? this will mean that anything NOT at local host WON'T get to:

```
     $scope.rooms.push({
        url: $scope.currentroom,
        key: $scope.currentroom.hashCode()
      });
```